engine/src/main/java/org/terasology/particles/updating/ParticleUpdater.java

```diff
@@ -45,9 +45,9 @@
      * @param registeredAffectorFunctions  The list of affector functions to use when processing the given system's affectors.
      * @param registeredGeneratorFunctions The list of generator functions to use when processing the given system's generators.
      */
-    void configureEmitter(final ParticleEmitterComponent emitter,
-                          final BiMap<Class<Component>, AffectorFunction> registeredAffectorFunctions,
-                          final BiMap<Class<Component>, GeneratorFunction> registeredGeneratorFunctions);
+    void configureEmitter(ParticleEmitterComponent emitter,
```

Comment:
I think the intention of adding the `final` keyword to the method parametes was: "The parameters for every implementation of this interface should be final".
Since this assumption is not correct, see: https://stackoverflow.com/questions/20317835/when-i-define-an-interface-methods-parameters-final-do-i-need-to-repeat-final-i it is redundat and also may be missleading if new implementations will be added in future

Reply:
Thanks for the clarification. You are correct, this should definitely be removed.